### PR TITLE
Feat/reduce hardcoding

### PR DESCRIPTION
**Pull request by sevesalm**
Osa diffeistä johtuu elm-formatista.

### CODE REVIEW

File: frontend/stylesheets/author-info.scss [hunk 1]
```diff
@@ -12,16 +12,16 @@
   }
   &__name {
     font-weight: bold;
-    @include gotham-bold;
-    color: $green;
+    @include display-font;
```
Comment (daniellandau) [inline frontend/stylesheets/author-info.scss]: It's more of a heading or highlight font

File: server/static/registerDescription.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[
```
Comment (daniellandau) [inline server/static/registerDescription.json]: Instead the format could be:

```
{ "heading': "whatever"
, "content": [
    { "heading": "whatever smaller, no number included"
    , "content": ["paragraph", "second paragraph"]
    },
    ...
  ]
}
```

Should be easier to parse in to the previous format.

Comment (ohanhi) [inline server/static/registerDescription.json]: @daniellandau How about we use Markdown for the entire content? That would enable us to have links etc. in the content.

**Comment by daniellandau** [comment]
Yeah I think markdown would make sense. It would also allow images for example, and it's only trusted users that can change the content.

**Pull request merged**